### PR TITLE
Add illuminati feature from r/place

### PR DESCRIPTION
**Pull request by haykam821**
In the original r/place, there was a button in the corner with an all-seeing eye on it. When pressed, this enabled "Illuminati mode", which automatically centered the board on the newest pixel placed.

With this pull request, the functionality has been brought to Pxls via a toggle in the settings menu.

### CODE REVIEW

File: resources/public/pxls.js [hunk 1]
```diff
@@ -1964,6 +1967,11 @@ window.App = (function () {
                     $("#audiotoggle").change(function () {
                         ls.set("audio_muted", this.checked);
                     });
+                    $("#illuminati")[0].checked = ls.get("illuminati");
```
Comment (TheRebelG0d) [inline resources/public/pxls.js]: When illuminati is enabled, the checkbox should be checked on page load.

File: resources/public/pxls.js [hunk 2]
```diff
@@ -1679,6 +1679,9 @@ window.App = (function () {
                     socket.on("pixel", function (data) {
                         $.map(data.pixels, function (px) {
                             board.setPixel(px.x, px.y, px.color, false);
+                            if (ls.get("illuminati")) {
+                                board.centerOn(px.x, px.y);
```
Comment (TheRebelG0d) [inline resources/public/pxls.js]: If the user enables this and then places a pixel, all open tabs will jump to that pixel, which shouldn't be happening. Probably needs an extra property on the event

Comment (haykam821) [reply resources/public/pxls.js]: > Probably needs an extra property on the event

What do you mean exactly?

File: resources/public/pxls.js (hunk 1 above)
Comment (haykam821) [reply resources/public/pxls.js]: What is this problem? For me, the checkbox is checked the same way before and after a refresh.

File: resources/public/pxls.js (hunk 2 above)
Comment (haykam821) [reply resources/public/pxls.js]: Ah, I'll try to implement this but I probably won't succeed since I'm not familiar with Java.

**Pull request closed without merging**